src/renderer/screens/lend/useCompoundAccountEnabled.js

```diff
@@ -0,0 +1,19 @@
+// @flow
+import type { AccountLike, Account } from "@ledgerhq/live-common/lib/types";
+import {
+  makeCompoundSummaryForAccount,
+  getAccountCapabilities,
+} from "@ledgerhq/live-common/lib/compound/logic";
+
+export default function useCompoundAccountEnabled(account?: AccountLike, parentAccount?: Account) {
+  if (!account || account.type !== "TokenAccount") return false;
+
+  // check if account already has lending enabled
+  const summary =
+    account.type === "TokenAccount" && makeCompoundSummaryForAccount(account, parentAccount);
+  const capabilities = summary
+    ? account.type === "TokenAccount" && getAccountCapabilities(account)
+    : null;
```

Comment:
The `account.type === "TokenAccount"` tests aren't needed here since done above.

Reply:
its for flow :sweat_smile: